styles/style.css

```diff
@@ -0,0 +1,2134 @@
+  color: #e0f2ff;
+}
+
+.products__title {
+  line-height: 1.75rem;
+  font-weight: 500;
+  text-align: center;
+  font-size: 1.25rem;
+}
+
+.list-item__image {
+  width: 14rem;
+  height: 14rem;
+  background-color: var(--card-background-color);
+  border-radius: var(--card-border-radius);
+  display: grid;
+}
+
+.list-item__icon-image {
+  width: 9.375rem;
+  height: 9.375rem;
+  place-self: center;
+  color: var(--primary-color);
+}
+
+.categories {
+  gap: 4rem;
+  align-items: center;
+}
+
+.categories > * {
+  width: 38rem;
+}
+
+.categories__list {
+  gap: 1rem;
+  justify-content: center;
+  position: relative;
+}
+
+.categories__list::after {
+  content: "";
+  position: absolute;
+  width: 50%;
+  top: 0;
+  right: 0;
+  min-height: 3.75rem;
+  background-image: linear-gradient(#fff, transparent);
+}
+
+.categories__list::before {
+  content: "";
+  position: absolute;
+  width: 50%;
+  bottom: 0;
+  left: 0;
+  min-height: 3.75rem;
+  background-image: linear-gradient(transparent, #fff);
+  z-index: 2;
+}
+
+.categories__categories-information {
+  order: 2;
+}
+
+.categories-information__description {
+  opacity: 0.7;
+}
+
+.categories-information__highlite-text {
+  width: 60%;
+}
+
+.categories-information__button {
+  margin-top: 1rem;
+}
+
+.categories__categories-cards {
+  flex-wrap: nowrap;
+  height: 28.75rem;
+  gap: 1.25rem;
+  overflow-x: auto;
+  position: relative;
+}
+
+.categories__categories-cards:last-child::before {
+  content: "";
+  width: 14rem;
+  min-height: 3.75rem;
+  background-color: var(--card-background-color);
+  border-radius: var(--card-border-radius);
+}
+
+.categories__categories-cards:first-child::after {
+  content: "";
+  width: 14rem;
+  min-height: 3.75rem;
+  background-color: var(--card-background-color);
+  border-radius: var(--card-border-radius);
+}
+
+.categories-cards__categories-card {
+  background-color: var(--card-background-color);
+  border-radius: var(--card-border-radius);
+  padding: 1.5rem;
+  transition: var(--color-transition);
+  gap: 0.5rem;
+  width: 10.8rem;
+}
+
+.categories-card__description {
+  color: var(--fade-color);
+  padding: 0;
+}
+
+.categories-card__image {
+  width: 3rem;
+  height: 3rem;
+  color: var(--primary-color);
+}
+
+.categories-card__title {
+  padding-top: 0.5rem;
+}
+
+.categories-cards__link:active .categories-cards__categories-card {
+  background-color: var(--primary-color);
+  color: #f8f7ff;
+}
+
+.categories-cards__link:active svg {
+  color: #e0f2ff;
+}
+
+.categories-cards__link:active .categories-card__description {
+  color: #eff5fad1;
+}
+
+.trending {
+  padding-bottom: 3.5rem;
+}
+
+.trending__trending-header {
+  justify-content: space-between;
+  align-items: end;
+}
+
+.trending-controller__buttons {
+  gap: 0.75rem;
+}
+
+.trending-controller__button:last-of-type svg {
+  transform: rotate(180deg);
+}
+
+.trending__trending-list {
+  margin-top: 3rem;
+  gap: 1rem;
+  overflow-x: auto;
+}
+
+.trending-card__to-product {
```

Comment:
3.12. ❤️ Use flex or grid for layouts.
Do not use tables or positioning.

The container(.trending-card__image-container) is already a Grid container, use it please.

Reply:
Not fixed